src/XrdHttp/XrdHttpMon.cc

```diff
@@ -13,9 +13,71 @@ typedef std::array<std::array<XrdHttpMon::HttpInfo, XrdHttpMon::StatusCodes::sc_
 
 StatsMatrix XrdHttpMon::statsInfo{};
 
-XrdHttpMon::XrdHttpMon(XrdSysLogger *logP, XrdXrootdGStream *gStream)
-    : flushPeriod(std::chrono::seconds(gStream->GetAutoFlush())), gStream(gStream)  {
-  eDest.logger(logP);
+bool XrdHttpMon::hasGStream = false;
+bool XrdHttpMon::hasMonRoll = false;
+
+RAtomic_uint XrdHttpMon::verbCounters[XrdHttpReq::ReqType::rtCount] = {0};
```

Comment:
By using V2 of monroll you can use a larger counter, say uint64 or ulong. This addresses @bbockelm concern of wrapping counters.